File: HackerRank-10-Days-Statistics/Day 5/Day_5_Normal_Distribution_II.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include<iomanip>
using namespace std;

double cum_dist(double x, double mean, double dev){
    double dist=0.5*(1+erf((x-mean)/(sqrt(2)*dev)));
    return dist;
}


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    double mean, dev, std_marks80, std_marks60;
    cin>>mean>>dev>>std_marks80>>std_marks60;
    
    cout<<fixed<<setprecision(2)<<(1-cum_dist(std_marks80,mean,dev))*100<<endl;
    cout<<fixed<<setprecision(2)<<(1-cum_dist(std_marks60,mean,dev))*100<<endl;
    cout<<fixed<<setprecision(2)<<(cum_dist(std_marks60,mean,dev))*100<<endl;
    return 0;
}

File: HackerRank-10-Days-Statistics/Day 5/Day_5_Normal_Distribution_II.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
import math
import sys

a,b,c = sys.stdin.readlines()
mean, dev = list(map(int,a.strip().split()))
std_marks80=int(b)
std_marks60=int(c)

def cum_dist(x):
    dist=0.5*(1+ math.erf((x-mean)/(dev*math.sqrt(2))))
    return dist

print(round((1-cum_dist(std_marks80))*100,2))
print(round((1-cum_dist(std_marks60))*100,2))
print(round(cum_dist(std_marks60)*100,2))
